### PR TITLE
Fixed cniVersion in the plugin configuration file - Issue #604

### PR DESCRIPTION
**Pull request by FlorianOtel**
*Issue #, if available:*

See #604 

*Description of changes:*

Added cniVersion string in the plugin configuration file

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: misc/10-aws.conflist [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "cniVersion": "0.3.0",
```
Comment (mogren) [inline misc/10-aws.conflist]: Thanks for adding this, just a minor nit, it should be:  `"0.3.1"`.

It's just a [small spelling fix in the spec from v0.3.0](https://github.com/containernetworking/cni/releases/tag/spec-v0.3.1), but we do return `ips` and not `ip`.

https://github.com/aws/amazon-vpc-cni-k8s/blob/566b0d44701a6c1bcd8e2c4022cc6cfc7b7c0c59/plugins/routed-eni/cni.go#L213

Comment (mogren) [reply misc/10-aws.conflist]: Changed it, and tested.

Comment (FlorianOtel) [reply misc/10-aws.conflist]: Was just about to bump the version in my repo then push the changes to my repo & re-submit 

Thanks !

**Comment by FlorianOtel** [comment]
Was just about to bump the version and push it to my repo to resolve this :)

Thanks for fixing this -- took several days for me to debug this issue. 

Will close #604 as well

**Comment by mogren** [comment]
@FlorianOtel Thank you for finding it. I was planning to get a 1.6.0-rc1 out soon, and I really wanted this fix in. 😄

**Pull request merged**